packages/use-codemirror/src/hooks/useCodeMirror.ts

```diff
@@ -397,9 +428,6 @@ function getCodeMirrorExtensions({
     extensions.push(jsonLinter)
   }
 
-  // Line numbers
-  if (lineNumbers) extensions.push(lineNumbersExtension())
```

Comment:
accident?

Reply:
changed the order to have the marker prepending the line numbers, otherwise it appends it